### PR TITLE
Ignore emoticon-like text

### PR DESCRIPTION
**Pull request by montlebalm**
This PR adds exceptions for emoticon-like text inside parsed strings. Prior to these changes `(around 8)` would be translated to `(around :sunglasses:`. Additionally the 8th item in a numbered list (i.e., `7)... 8)...`) would also get the "sunglasses" treatment.

I've written tests for the known edge cases, and I'm happy add any more we need.

### CODE REVIEW

File: build/build.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?
+<?php
```
Comment (montlebalm) [inline build/build.php]: Required to fix build process.

**Comment by iamcal** [comment]
Merged in #50
